patched_bdist_wheel.py

```diff
@@ -0,0 +1,103 @@
+"""
```

Comment:
Perhaps it needs a better name, but I had thought `integration` would be for all build-related stuff.  `tools` is really developer tools.